docs/subnets/deploy-a-gnosis-safe-on-your-evm.md

```diff
@@ -0,0 +1,810 @@
+Nonce: 1658256419254
+To (factory): 0x17aB05351fC94a1a67Bf3f56DdbB941aE6c63E25
+Data: 0x1688f0b900000000000000000000000095ca0a568236fc7413cd2b794a7da24422c2bbb600000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000018217c8e9b60000000000000000000000000000000000000000000000000000000000000164b63e800d0000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000140000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000008db97c7cece249c2b98bdc0226cc4c2a57bf52fc000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
+```
+
+
+Notice the line, "_Deploy Safe to 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114_", informs us that our safe contract lives at the address `0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114`. For demonstration purposes, we will utilize this address For this section of the article.
+
+Lets inspect our Safe details by running the following:
+
+```zsh
+yarn safe info --network subnet 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114 
+```
+
+Output:
+```zsh
+Checking Safe at 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114
+Singleton: 0x95CA0a568236fC7413Cd2b794A7da24422c2BBb6
+Version: 1.3.0
+Owners: 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC
+Threshold: 1
+Nonce: 0
+Fallback Handler: 0x0000000000000000000000000000000000000000
+Modules: 
+```
+
+The output above illustrates a few things:
+- `Singleton` - This is the contract that holds the logic for our safe to interact with. In this case, the Smart Contract we are using is [`GnosisSafeL2.sol`](https://github.com/safe-global/safe-contracts/blob/main/contracts/GnosisSafeL2.sol), A multisignature wallet with support for confirmations using signed messages based on [ERC191](https://eips.ethereum.org/EIPS/eip-191).
```

Comment:
Be consistent when referring `safe` as `Safe`. Please search and correct all occurrences.